assets/js/vue/FileManager.vue

```diff
@@ -25,6 +15,46 @@
                 </p>
             </template>
         </DxPopup>
+        <DxPopup
+            title="Uploading Files"
+            :visible.sync="loadingVisible"
+            :close-on-outside-click="false"
+            :show-title="false"
+            position="center"
+            :showCloseButton="false"
+            :width="300"
+            :height="250"
+            :drag-enabled="false"
+        >
+            <template>
+                <center>
```

Comment:
<center> tag is deprecated

Reply:
I will become a div with a class and styling.